e2e/e2e_test.go

```diff
@@ -106,24 +95,21 @@ var _ = Describe("e2e", func() {
 			selector = selector.Add(*reqCpRole, *reqMRole)
 			Expect(k8sClient.List(context.Background(), workers, &ctrl.ListOptions{LabelSelector: selector})).ToNot(HaveOccurred())
 			Expect(len(workers.Items)).To(BeNumerically(">=", 3))
-			node1 = &workers.Items[0]
```

Comment:
out of curiosity, what was the reason to shift the nodes under test, skipping node 0?

Reply:
good question... can't remember 😆 Maybe I just though that there are potentially less workloads on node 1 and 2 to move around 🤔 Not sure if that would make sense though 🤷🏼‍♂️

Reply:
If there is no particular reason to choose one node over another, I wonder if making the selection random would make sense.

Reply:
not worth the effort IMHO 🤔